docs/.storybook/configs/intl-context.js

```diff
@@ -25,7 +27,9 @@ const getMessagesForLocale = (locale) => {
 const namifyLocale = (locale) => {
   switch (locale) {
     case 'en':
-      return 'English';
+      return 'English (US)';
+    case 'en-GB':
+      return 'English (British)';
```

Comment:
I added british english to Storybook so we can play also with that locale, although it loads the same translations that american english.